plugin/forward/persistent.go

```diff
@@ -141,6 +147,20 @@ func (t *transport) cleanup(all bool) {
 	}
 }
 
+func (t *transport) dialTimeout() time.Duration {
```

Comment:
Can this be put in connect.go next to the avg calculation? That will probably flag some similarities that could be factored out a bit.